libs/features/mas/commerce/src/settings.js

```diff
@@ -135,15 +133,18 @@ function getSettings(config = {}) {
     // TODO: add alias names for meta, search and storage
     // See https://git.corp.adobe.com/wcms/tacocat.js/pull/348#discussion_r6557570
     const { commerce = {}, locale = undefined } = config;
-    const hostEnv =
-        config.env?.name === HostEnv.PROD
-            ? HostEnv.PROD
-            : toEnumeration(
-                  getParameter(PARAM_ENV, commerce, { metadata: false }),
-                  HostEnv,
-                  HostEnv.PROD,
-              );
-    const env = hostEnv === HostEnv.STAGE ? Env.STAGE : Env.PRODUCTION;
+    const lowHostEnv = ['local', 'stage'].includes(config.env?.name);
+    let env = Env.PRODUCTION;
+    let wcsURL = 'https://www.adobe.com/web_commerce_artifact';
+    if (lowHostEnv) {
+      wcsURL = 'https://www.stage.adobe.com/web_commerce_artifact';
```

Comment:
is this because authority of lower env is stage? that would be a good reason to do this.

Reply:
not really no, it works also with `www.adobe.com`
i think our authority theory is wrong.

but I did use www.stage.adobe.com for lower envs just to be consistent with how rest of milo works, i think it would be more confusing for Devs to see 'prod' commerce URL on their local while they see stage one on stage.

Reply:
fixed to be www.adobe.com